game/graphics/mesh/pose.cpp

```diff
@@ -812,7 +812,7 @@ Vec2 Pose::headRotation() const {
   return Vec2(headRotX,headRotY);
   }
 
-void Pose::setAnimRotate(const AnimationSolver &solver, Npc &npc, WeaponState fightMode, int dir) {
+void Pose::setAnimRotate(const AnimationSolver &solver, Npc &npc, WeaponState fightMode, enum TurnAnim turn, int dir) {
```

Comment:
> `enum TurnAnim turn`

In opengothic we are not using `enum` or `struct` as part of function signature. In C++ you don't  need to.

Reply:
done too